docs/CNI/calico.md

```diff
@@ -424,7 +424,7 @@ calico_wireguard_enabled: true
 
 The following OSes will require enabling the EPEL repo in order to bring in wireguard tools:
 
-* CentOS 7 & 8
+* CentOS 8
```

Comment:
All versions of CentOS are already EOL, do I think it's possible to remove it completely?